tests/report/junit/data/custom.xml.j2

```diff
@@ -0,0 +1,3 @@
+{% for result in RESULTS %}
+name='{{ result.name | e }}',value='{{ result.result.value | e }}'
+{% endfor %}
```

Comment:
I would rather try a simple XML - not emitting an XML could put a thought in users' heads that using this plugin to render anything but XML is a good idea :)

Reply:
I modified the test to use the XML. Thanks.